docs/variants/alder-lake-workstation/test-matrix.md

```diff
@@ -0,0 +1,60 @@
+# Test matrix - Alder Lake Workstation
+
+## About
+
+<!--
+The test matrix is used to determine which of the test suites and test cases
+described in this documentation are dedicated to the given platform
+-->
+
+## Module: Dasharo compatibility
+
+| No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
+|:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
+| 1.   | [UEFI compatible interface][EFI]                  | EFI           | EFI001.001                           |
+| 2.   | [UEFI Shell][USH]                                 | USH           | All                                  |
+| 3.   | [Display ports and LCD support][DSP]              | DSP           | DSP002.001, DSP002.003, DSP003.001, DSP003.003 |
+| 4.   | [USB HID and MSC Support][USB]                    | USB           | USB001.001, USB001.002, USB002.001, USB002.002 |
+
+[EFI]: ../../unified-test-documentation/dasharo-compatibility/30M-uefi-compatible-interface.md
+[USH]: ../../unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md
+[DSP]: ../../unified-test-documentation/dasharo-compatibility/31E-display-ports-and-lcd.md
+[USB]: ../../unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md
+
+<!--
```

Comment:
Why is this commented out? IS this supposed to be included in the table above with later phases?

Reply:
Yes, the commented-out tables contain the full testing scope. When we reach the later project stages, we will add this test suites to the visible part.